slave/vagrant/fabfile.py

```diff
@@ -0,0 +1,56 @@
+from fabric.api import run, task, sudo, put, local
+from twisted.python.filepath import FilePath
+from StringIO import StringIO
+import yaml
+
+# WARNING: Although this code has been tested to produce a working slave
+# the production slave wasn't configured with this code.
+
+def configure_gsutil():
+    boto_config = FilePath(__file__).sibling('boto-config.in').getContent()
+    output = local('lpass show --notes "google-cert@build.clusterhq.com"',
+                   capture=True)
+    cert = yaml.safe_load(output.stdout)
```

Comment:
lastpass needed updating. lastpass (at least the command-line client) doesn't like embedded nulls, which the certificate has. Also, there are two pieces of data that need storing.